protoconvert.go

```diff
@@ -65,6 +65,9 @@ func valueToProto(value any) (*pb.Value, error) {
 	case net.IP:
 		return &pb.Value{Inner: &pb.Value_Inet{Inet: &pb.Inet{Value: v[:]}}}, nil
 	case *uuid.UUID:
+		if v == nil {
+			return &pb.Value{Inner: &pb.Value_Null_{Null: &pb.Value_Null{}}}, nil
```

Comment:
It might make sense to create a global value (constant) for a null value:

```go
var nullValue = &pb.Value{Inner: &pb.Value_Null_{Null: &pb.Value_Null{}}}

	case *uuid.UUID:
		if v == nil {
			return nullValue, nil
```